src/javascript/app/pages/trade/common.js

```diff
@@ -389,6 +389,31 @@ const commonTrading = (() => {
         };
     };
 
+    /*
+     * Function is called only once each ${wait} seconds
+     * last call is debounced
+     */
+    const throttlebounce = (func, wait = 500) => {
```

Comment:
Please move this function to `javascript/_common` folder, since it's not dependent and it could be used in other sections.
Thanks.